integration_test/third_party_apps_data/applications/couchbase/debian_ubuntu/install

```diff
@@ -5,11 +5,11 @@ set -e
 sudo apt-get update
 sudo apt-get install -y gnupg bzip2 wget curl libncurses5
 
-oscode="$(lsb_release -sc)"
-if [ "$oscode" = "bullseye" ]; then
-  # currently debian 11 is not supported, install buster package instead
+source /etc/os-release
+if [[ "${VERSION_CODENAME}" = "lunar" || "${VERSION_CODENAME}" == "jammy" ]]; then
+  # currently ubuntu 23.04 (lunar) and 22.04 (jammy) are not supported, install focal (ubuntu 20.04) package instead
```

Comment:
Our customers aren't going to do that, are they? Should we just disable CouchBase support on Ubuntu 23.04 (and 22.04) instead?

Reply:
For this and all the other comments re: disabling support on Ubuntu 23.04, there's past precedent for this pattern of installing a previous package version for technologies that don't support more current OS releases. For example, see this file - I removed the behavior that installed the `buster` package on `bullseye` (since CouchBase now works on `bullseye`), but that behavior is a perfect example of the pattern.